testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -122,14 +126,15 @@ private DockerContainer createTestsContainer()
         DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/centos7-oj8:" + imagesVersion, TESTS)
                 .withCopyFileToContainer(forHostPath(dockerFiles.getDockerFilesHostPath()), "/docker/presto-product-tests")
                 .withCommand("bash", "-xeuc", "echo 'No command provided' >&2; exit 69")
+                .withEnv("JAVA_HOME", javaHome)
                 .waitingFor(new WaitAllStrategy()) // don't wait
                 .withStartupCheckStrategy(new IsRunningStartupCheckStrategy());
 
         return container;
     }
 
     @SuppressWarnings("resource")
-    public static DockerContainer createPrestoContainer(DockerFiles dockerFiles, File serverPackage, boolean debug, String dockerImageName, String logicalName)
+    public static DockerContainer createPrestoContainer(DockerFiles dockerFiles, File serverPackage, String javaHome, boolean debug, String dockerImageName, String logicalName)
```

Comment:
nit: would consider passing `SupportedTrinoJdk` into this method

Reply:
Done.